modules/metaphlan3/metaphlan3/main.nf

```diff
@@ -9,7 +9,7 @@ process METAPHLAN3 {
 
     input:
     tuple val(meta), path(input)
-    path metaphlan_db
+    path metaphlan_bt2_db
```

Comment:
the yml probably needs some renaming for this one

Reply:
Wasn't even in hte YAML in the first place :scream: